src/utils/SetupUtil.js

```diff
@@ -122,6 +114,8 @@ export default {
           throw new Error('Could not determine IP from docker-machine.');
         }
 
+        if (util.isLinux()) { docker.ensureLinuxDockerIsRunning() }
```

Comment:
And this is could be moved in the `true` part of the if.

Finally, what we could, in a proactive way, is to have a function to configure Docker in a native way (Linux and soon Windows), and a second function for the non-native way (OSX and ...?).

What do you think about that ?